### PR TITLE
Haddocks

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: scripts/haddocs.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Build haddock documentation and an index for all projects in
+# `ouroboros-network` repository.
```
Comment (Jimbo4350) [inline scripts/haddocs.sh:3]: ouroboros-network leftover here

File: scripts/haddocs.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  for package in $(ls "${OUTPUT_DIR}"); do
+    echo "--read-interface=${package},${OUTPUT_DIR}/${package}/${package}.haddock"
+  done
+}
+
+# Generate top level index using interface files
+#
+haddock \
+  -o ${OUTPUT_DIR} \
+  --title "ouroboros-network" \
```
Comment (Jimbo4350) [inline scripts/haddocs.sh:76]: ouroboros-network left over here

File: scripts/prolog [hunk 3]
```diff
@@ -0,0 +1,65 @@
+= Ouroboros-Network Repository Hackage Documentation
```
Comment (Jimbo4350) [inline scripts/prolog:1]: Ouroboros-network left over here

File: scripts/haddocs.sh (hunk 2 above)
Comment (newhoggy) [reply scripts/haddocs.sh]: Fixed.  Thanks!

File: scripts/haddocs.sh (hunk 1 above)
Comment (newhoggy) [reply scripts/haddocs.sh]: Fixed.  Thanks!

File: scripts/prolog (hunk 3 above)
Comment (newhoggy) [reply scripts/prolog]: Fixed.  Thanks!

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**